BackofficeBundle/DependencyInjection/OpenOrchestraBackofficeExtension.php

```diff
@@ -67,6 +67,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('authorize_status_change.yml');
         $loader->load('authorize_edition.yml');
         $loader->load('restore_entity.yml');
+        $loader->load('dashboard.yml');
```

Comment:
@nicolasThal : If the integrator alters his app/config/parameters.yml, his settings replace the Orchestra default ones. Do you think we should provide another way to override that parameter ?